src/databricks/labs/ucx/install.py

```diff
@@ -169,11 +176,43 @@ def run(
             self._product_info,
         )
         try:
-            workspace_installation.run()
+            workspace_installation.run(self._is_account_install, config.trigger_job)
```

Comment:
```suggestion
            workspace_installation.run()
```
Config is already injected via constructor, inject the property about account-level install